src/views/Farms/components/FarmTable/Actions/StakedAction.tsx

```diff
@@ -130,7 +130,7 @@ const Staked: React.FunctionComponent<StackedActionProps> = ({
             width="100%"
             onClick={onPresentDeposit}
             variant="secondary"
-            disabled={location.pathname.includes('archived')}
+            disabled={location.pathname.includes('history') || location.pathname.includes('archived')}
```

Comment:
I think `['history', 'archived'].includes(location.pathname)` would be cleaner.

Reply:
That doesn't work, but i changed to 

`["history", "archived"].some(item => location.pathname.includes(item))`